pkgs/development/libraries/qt-6/modules/qtdeclarative.nix

```diff
@@ -3,16 +3,24 @@
 , qtlanguageserver
 , qtshadertools
 , openssl
+, stdenv
 , python3
+, lib
+, pkgsBuildBuild
 }:
 
 qtModule {
   pname = "qtdeclarative";
-  propagatedBuildInputs = [ qtbase qtlanguageserver qtshadertools openssl python3 ];
+  propagatedBuildInputs = [ qtbase qtlanguageserver qtshadertools openssl ];
+  nativeBuildInputs = [ python3 ];
```

Comment:
This is the line that triggers the mass rebuild. I don't think python 3 is is supposed to build a buildInput